packages/toolkit/src/view/model/view-settings/ModelSettingsContentViewer.tsx

```diff
@@ -0,0 +1,39 @@
+import { ModelSettingsEditForm, ModelSettingsVersions } from ".";
+import { Model } from "../../../lib";
+import { ModelSettingsTabs } from "./ModelSettingsHead";
+
+export type ContentViewerProps = {
+  selectedTab: ModelSettingsTabs;
+  model?: Model;
+  onUpdate: () => void;
+};
+
+export const ModelSettingsContentViewer = ({
+  selectedTab,
+  model,
+  onUpdate,
+}: ContentViewerProps) => {
+  if (!model) {
+    return null;
+  }
```

Comment:
@orangecoloured If the model is not there, is it due to the query is still loading?

Reply:
I updated the code to show spinner while the model is loading